src/vivarium_gates_nutrition_optimization_child/components/wasting.py

```diff
@@ -23,8 +24,21 @@
 from vivarium_gates_nutrition_optimization_child.utilities import get_random_variable
 
 
-class ChildWasting:
+class ChildWasting(Component):
+    @property
+    def columns_required(self) -> Optional[List[str]]:
+        return ["alive", "age", self.dynamic_model.state_column]
```

Comment:
Aren't you missing `self.static_model.propensity_column_name`?

Reply:
That's specified in the initialization requirements

Reply:
Actually, the `ChildWasting` component ought not to have any initialization requirements, since it doesn't have an `on_initialize_simulants` method. 

My comment was based solely off of looking at that property and not seeing that column in the columns required. All columns required during a component's initialization are by definition required by the component.

Reply:
I removed the initialization requirements and added the static propensity column name and the sim runs.